pkg/suggestion/v1alpha3/chocolate/base_chocolate_service.py

```diff
@@ -32,12 +32,8 @@ def getSuggestions(self, search_space, trials, request_number):
                 chocolate_search_space[key] = choco.quantized_uniform(
                     int(param.min), int(param.max), 1)
             elif param.type == DOUBLE:
-                if param.step != None:
-                    chocolate_search_space[key] = choco.quantized_uniform(
-                        float(param.min), float(param.max), float(param.step))
-                else:
-                    chocolate_search_space[key] = choco.uniform(
```

Comment:
what was the default value here?

Reply:
Validation will fail without step.

Reply:
I meant, what does the algorithm assume if step is not set?

Reply:
The algorithm will return validation error if step is not set